include/cache.hpp

```diff
@@ -67,7 +69,7 @@ class fixed_sized_cache
 
     const Value &Get(const Key &key) const
     {
-        operation_guard lock{safe_op};
+        read_guard lock{safe_op};
```

Comment:
not thread-safe
Mutex here doesn't help.
Either remove the mutex or return by value.

Reply:
Ok, fixed in 6fcad82